examples/typescript-ex/src/index.ts

```diff
@@ -1,45 +1,48 @@
-import { CloudEvent, HTTPREceiver } from '../../';
+import { CloudEvent, HTTPReceiver } from '../../..';
+import { CloudEventV1 } from '../../../lib/v1';
```

Comment:
Before this lands, we should release 2.0.2 and change these to actually import the published module

Reply:
+1